utils/downloads.py

```diff
@@ -31,18 +32,19 @@
 from sounds.models import License
 
 
-def download_sounds(sounds_list, sounds_url=None):
+def download_sounds(sounds_list, licenses_url, pack):
     """
     From a list of sounds generates the HttpResponse with the information of
     the wav files of the sonds and a text file with the license. This response
     is handled by mod_zipfile of nginx to generate a zip file with the content.
     """
     users = User.objects.filter(sounds__in=sounds_list).distinct()
     # Generate text file with license info
-    licenses = License.objects.all()
+    license_ids = sounds_list.values('license_id').distinct()
+    licenses = License.objects.filter(id__in=license_ids).all()
     attribution = render_to_string("sounds/pack_attribution.txt",
```

Comment:
We get this data and render the template twice in this code, once here and once in the `pack_licenses` view. Should we have a util method (as part of the `Pack` class?) so that we only have to make changes in one place?